src/applications/vaos/new-appointment/components/DateTimeSelectPage/WaitTimeAlert.jsx

```diff
@@ -60,45 +109,37 @@ export const WaitTimeAlert = ({
               </li>
               <li>Call your local VA medical center</li>
             </ul>
-            <p className="vads-u-font-weight--bold">
-              If you have an urgent medical need or need care right away:
-            </p>
+            <UrgentCareMessage />
+            <ActionButtons
+              eligibleForRequests={eligibleForRequests}
+              facilityId={facilityId}
+              onClickRequest={onClickRequest}
+            />
+          </>
+        </InfoAlert>
+      );
+    } else {
```

Comment:
This specifies for no slots available plus will account for both same day (urgent care) and non-same day appts.
```suggestion
    } else if (!hasNextAvailableApptDate) {
```

Reply:
Good catch! Done.